app/scripts/modules/core/src/account/account.service.ts

```diff
@@ -152,29 +147,11 @@ export class AccountService {
       });
   }
 
-  public getUniqueGceZonesForAllAccounts(provider: string): ng.IPromise<IZone> {
```

Comment:
@danielpeach I do not see any references to this method in the codebase so I'm removing it. Let me know if I'm missing something.